### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by OlenaVdovychenko**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -8,6 +8,30 @@ describe(`Function 'getHumanAge':`, () => {
...
+
+  it(`getHumanAge(5, 7) should return [0, 0]`, () => {
+    expect(getHumanAge(5, 7)).toMatchObject([0, 0]);
+  });
+
+  it(`getHumanAge(34, 49) should return [4, 7]`, () => {
+    expect(getHumanAge(34, 49)).toMatchObject([4, 7]);
+  });
+
+  it(`input: catAge = 0, dogAge= 0 should return [0, 0]`, () => {
```
Comment (Danylo25) [inline src/getHumanAge.test.js:26]: Думаю, що так добре, але краще було б якби однакоко було, тобто або в такому форматі getHumanAge(catAge, dogAge) або в такому input: catAge = 0, dogAge= 0.

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -8,6 +8,30 @@ describe(`Function 'getHumanAge':`, () => {
   });
```
Comment (brainwashers) [inline src/getHumanAge.test.js:9]: давай чекнимо всі граничні значення по котам і собакам

**Review by brainwashers — CHANGES_REQUESTED** [review]

File: src/getHumanAge.test.js (hunk 2 above)
Comment (OlenaVdovychenko) [reply src/getHumanAge.test.js:9]: @brainwashers додала тести по граничних значеннях (commit: correct tests for boundary values were added)
